app/lib/features/chat_ng/widgets/events/profile_update_event.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:acter/common/extensions/options.dart';
+import 'package:acter/common/providers/common_providers.dart';
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/common/utils/room_member.dart';
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter/l10n/generated/l10n.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show ProfileContent, TimelineEventItem;
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class ProfileUpdateEvent extends ConsumerWidget {
+  final bool isMe;
+  final String roomId;
+  final TimelineEventItem item;
+
+  const ProfileUpdateEvent({
+    super.key,
+    required this.isMe,
+    required this.roomId,
+    required this.item,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    String textMsg = getStateEventStr(context, ref, item) ?? '';
+
+    return Container(
+      padding: const EdgeInsets.only(left: 10, bottom: 5, right: 10),
+      child: RichText(
+        text: TextSpan(
+          text: textMsg,
+          style: Theme.of(context).textTheme.labelSmall,
+        ),
+      ),
+    );
+  }
+
+  String? getStateEventStr(
+    BuildContext context,
+    WidgetRef ref,
+    TimelineEventItem item,
+  ) {
+    final lang = L10n.of(context);
+    final myUserId = ref.read(myUserIdStrProvider);
+    ProfileContent content = item.profileContent().expect(
```

Comment:
don't do `expect` in build code, it can runtime crash the app.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2790/commits/69e0f58102392c3f9065aae26961522382532bbe